obs_status.yml

```diff
@@ -1,4 +1,5 @@
 bad:
+  76813072: 10
```

Comment:
in the other cases of bad position stars, you also changed the comments.

Reply:
Comment updated